.appveyor.yml

```diff
@@ -1,17 +1,18 @@
 environment:
   matrix:
-    - TARGET: x86_64-pc-windows-msvc
-    - TARGET: i686-pc-windows-msvc
     - TARGET: x86_64-pc-windows-gnu
       MSYS_BITS: 64
-    - TARGET: i686-pc-windows-gnu
-      MSYS_BITS: 32
 
 install:
   - ps: Start-FileDownload "https://static.rust-lang.org/dist/rust-nightly-${env:TARGET}.exe"
   - rust-nightly-%TARGET%.exe /VERYSILENT /NORESTART /DIR="C:\Program Files (x86)\Rust"
   - set PATH=%PATH%;C:\Program Files (x86)\Rust\bin
-  - if defined MSYS_BITS set PATH=%PATH%;C:\msys64\mingw%MSYS_BITS%\bin
```

Comment:
MSYS2 GCC needs `__acrt_iob_func` but Rust shipped GCC doesn't like it, try without this line.